scanner/src/main/kotlin/LocalScanner.kt

```diff
@@ -167,6 +180,26 @@ abstract class LocalScanner(name: String, config: ScannerConfiguration) : Scanne
         }
     }
 
+    private fun getResultsFile(scannerDetails: ScannerDetails, pkg: Package, outputDirectory: File): File {
+        val scanResultsForPackageDirectory = File(outputDirectory, pkg.id.toPath()).apply { safeMkdirs() }
+        return File(scanResultsForPackageDirectory, "scan-results_${scannerDetails.name}.$resultFileExt")
+    }
+
+    private fun readFromStorage(scannerDetails: ScannerDetails, pkg: Package, outputDirectory: File): List<ScanResult> {
+        val resultsFile = getResultsFile(scannerDetails, pkg, outputDirectory)
+
```

Comment:
Nit: Remove this blank line (even if it was there before)?

Reply:
Done.